packages/video_player/lib/video_player.dart

```diff
@@ -132,6 +132,15 @@ class VideoPlayerValue {
 
 enum DataSourceType { asset, network, file }
 
+class VideoPlayerOptions {
+  VideoPlayerOptions({this.mixWithOthers = false});
+
+  /// set this to true, to mix the video players audio with other audio
+  /// sources. The default for iOS is to not mix the audio. The default for
```

Comment:
Isn't this description of the defaults confusing, because in this library both platforms will have `mix with others` set to `false` (line 136).

I think either the default should be platform specific or the sentence rephrased that it regards the native platforms behavior, but not the setting in here.

Reply:
I agreed! I think we can only say 
`Set this to true to mix the video players audio with other audio sources. The default value is false.`